src/foundry/client-esm/applications/apps/compendium-art-config.d.mts

```diff
@@ -1,4 +1,39 @@
+import type { ApplicationConfiguration, ApplicationRenderContext, ApplicationRenderOptions } from "../_types.d.mts";
 import type ApplicationV2 from "../api/application.d.mts";
 import type HandlebarsApplicationMixin from "../api/handlebars-application.mjs";
 
-export default class CompendiumArtConfig extends HandlebarsApplicationMixin(ApplicationV2) {}
+/**
+ * An application for configuring compendium art priorities.
+ */
+declare class CompendiumArtConfig extends HandlebarsApplicationMixin(ApplicationV2) {
+  static override DEFAULT_OPTIONS: Partial<ApplicationConfiguration>;
+  static override PARTS: Record<string, HandlebarsApplicationMixin.HandlebarsTemplatePart>;
+
+  /* -------------------------------------------- */
+  /*  Rendering                                   */
+  /* -------------------------------------------- */
+
+  /** @override */
+  override _prepareContext(_options: ApplicationRenderOptions): Promise<ApplicationRenderContext>;
+
+  /* -------------------------------------------- */
+  /*  Event Handlers                              */
+  /* -------------------------------------------- */
+
+  /**
+   * Adjust the priority of a package.
+   * @param _event          - The click event.
+   * @param target          - The button that was clicked.
+   */
+  static #onAdjustPriority(_event: MouseEvent, target: HTMLButtonElement): Promise<void>;
```

Comment:
We don't type private properties and functions because there's no way for downstream consumers to interact with them.